tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/matter/createMatter.controller.js

```diff
@@ -910,7 +948,20 @@
                         break;
                     case 3:
                         cm.prevButtonDisabled = false;
-                        cm.nextButtonDisabled = true;
+                        cm.nextButtonDisabled = true; cm.nextButtonDisabled = false;

```

Comment:
logic to make next and previous button enable disable as per new tab for extra fields is visile or not.